### ISSUE TITLE
Text guidance

### ISSUE THREAD

**Issue by Anastasia0411** [rinongal/textual_inversion#123]
Hello. I've faced the following issue: I use stable diffusion model (checkpoint sd-v1-4 from https://github.com/CompVis/stable-diffusion) to introduce this concept 
![image](https://user-images.githubusercontent.com/47336656/205281264-74e32227-bf3d-4ddf-b3bd-220c12caf6ca.png)
Unfortunately, during inference time the model ignores all additional words in my prompt. It generates just this teapot and nothing else. For example for the prompt "a * is floating in the sea", I got
![image](https://user-images.githubusercontent.com/47336656/205282255-3c49a615-10f3-4196-b6ad-487fc26f9d4c.png)
This teapot looks good, but there is no information about sea on this image. I tried different guidance scales with no positive effect. Has someone else faced this problem as well?
Thank you

**Comment by rinongal** [comment]
Sorry for the late response. 

How many vectors are you using to represent your concept? And what learning rate? You might be overfitting the concept.
